reference/interbit-ui-components/README.adoc

```diff
@@ -0,0 +1,85 @@
+= Interbit UI Components
+
+A library of React UI components specifically designed for use with Interbit.
+
+Interbit UI Components is intended to help developers with the frontend UI
+portion of developing Interbit applications.
+
+
+== Getting Started
+
+There are two ways to get started with Interbit UI Components. The easiest is
+to use the Interbit Template app as a starting point. You can also add Interbit
```

Comment:
Using `interbit-template` is not really a starting point for Interbit UI Components. It is a starting point for building an Interbit application that happens to use Interbit UI Components. Maybe something like:

'The easiest is to use the Interbit Template app as a starting point for a new Interbit application and use Interbit UI Components to build the applications UI.'

Reply:
👌